src/method_info.cpp

```diff
@@ -0,0 +1,19 @@
+#include "../include/method_info.hpp"
+
+MethodInfo::MethodInfo(ClassFile *class_file) {
+    this->class_file = class_file;
+    this->class_file->file_reader->readBytes(2, this->access_flags);
+    this->class_file->file_reader->readBytes(2, this->name_index);
+    this->class_file->file_reader->readBytes(2, this->descriptor_index);
+    this->class_file->file_reader->readBytes(2, this->attributes_count);
+    AttributeInfo::loadAttributes(this->attributes, 1, this->class_file);
```

Comment:
Aquele 1 hard coded tava la pra testar, antes de juntar na master troca pra this->attributes_count